### ISSUE TITLE
Is NFT Marker working in mobile?

### ISSUE THREAD

**Issue by densetsughem** [artoolkit/jsartoolkit5#56]
I tried using the sample loadNFTMArker but is it working in mobile?

arController.loadNFTMarker('nft-digify/data/pinball', function(markerId) {
			var markerRoot = arController.createThreeNFTMarker(markerId);
			alert(markerId);
		//	document.getElementById("mm").innerHTML = "LOAD NFT";

			console.log(markerRoot);
			
			markerRoot.add(sphere);
			arScene.scene.add(markerRoot);
		});

The code didn't detect the pinball marker. 
thanks.

**Comment by D504** [comment]
As I see NFT Marker does not supported yet.

https://github.com/artoolkit/jsartoolkit5/blob/master/doc/using_different_marker_types.html#L107

**Comment by letoast** [comment]
No, that's not true. That document was modified in 2015.

There is actually a branch of jsArtoolkit5 called NFT, where NFT works. Try it on desktop:
https://rawgit.com/artoolkit/jsartoolkit5/nft/examples/nft_threejs.html

The problem is that it doesn't work on mobile and I suspect the reason being: the way it tries to initialize the camera hasn't been updated to the way the master fork does it. Other examples in the "nft" fork don't work on mobile as well, while other examples in the "master" fork DO work on mobile.

**Comment by benebol** [comment]
I don't understand where the api gets the marker from:
```
		arController.loadNFTMarker('DataNFT/pinball', function(markerId) {
			var markerRoot = arController.createThreeNFTMarker(markerId);
			markerRoot.add(sphere);
			arScene.scene.add(markerRoot);
		});
```
DataNFT/pinball isn't a physic file? How can I replace this marker by a custom marker?
Thanks in advance.

**Comment by ThorstenBux** [comment]
@benebol the API you are referring too is in regards to NFT tracking. Which branch/fork are you using as NFT tracking never made it into the ARToolKit or artoolkitX repository.

**Comment by benebol** [comment]
@ThorstenBux There is a working example over [here](https://rawgit.com/artoolkit/jsartoolkit5/nft/examples/nft_threejs.html).

I downloaded all the content, but there is no folder named "DataNFT" which I should expect...

**Comment by ThorstenBux** [comment]
Can you also point me to the github repository where that come from?
My initial guess is that someone simply forgot to commit the DataNFT directory. Does it track with the pinball marker? You can get it here: https://github.com/artoolkitx/artoolkitx/blob/master/Examples/2d%20tracking%20example/Android/AR2DTracking_Proj/AR2DTrackingExample/src/main/assets/Data/pinball.jpg

**Comment by benebol** [comment]
Yes that's the weird part, the script is actually working, but the folder isn't their... 
You can simply visit the demo URL mentioned above and scan the pinball image with your camera. 
I can't find the Github repository either.

**Comment by ThorstenBux** [comment]
https://github.com/artoolkit/jsartoolkit5/tree/nft/examples there you go. And the DataNFT folder is there :)

**Comment by benebol** [comment]
@ThorstenBux Thank you very much! Im going to create a custom marker and replace the 3d model this weekend!

**Comment by DazHaze** [comment]
nft on mobile is very slow in my experince, has anyone found a way to optimize it ?

**Comment by tomasdev** [comment]
when will JSARToolkit support NFT markers?

**Comment by dingxiaowei1995** [comment]
when will JSARToolkit support NFT markers?
